test/unit/test_arguments.py

```diff
@@ -310,7 +300,21 @@ def test_tutorial_averaging_args(test_params, expected_params, expected_params_p
     _test_args_subset(test_params, expected_params, expected_params_present, arguments.add_average_args)
 
 
-def _create_argument_values_that_must_be_files(params):
+@pytest.mark.parametrize("test_params, expected_params", [
+    ('--source test_src --target test_tgt --output prepared_data ',
+     dict(source='test_src', target='test_tgt',
+          source_vocab=None, target_vocab=None, shared_vocab=False, num_words=(50000, 50000), word_min_count=(1,1),
```

Comment:
can we format to 1 argument per line? This makes diffs look a lot cleaner in the future

Reply:
sure, done